index.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+﻿<!DOCTYPE html>
```

Comment:
There is no changes here, still GitHub detecting some changes. @suhailsinghbains buggy GitHub